share/models/creative.py

```diff
@@ -1,16 +1,23 @@
+import logging
+
 from django.db import models
+from django.db import transaction
 from django.utils.translation import ugettext_lazy as _
 
 from share.models.base import ShareObject
 from share.models.base import TypedShareObjectMeta
 from share.models.meta import Tag, Subject
+from share.models.change import Change
 from share.models.fields import ShareManyToManyField, ShareURLField
 
 from share.util import strip_whitespace, ModelGenerator
 
 
+logger = logging.getLogger(__name__)
+
 # Base Creative Work class
 
+
```

Comment:
Probably both of the newlines between the comment and the class should be deleted.